src/Tools/Source/SyntaxVisualizer/SyntaxVisualizerExtension/project.json

```diff
@@ -7,21 +7,23 @@
     "Microsoft.CodeAnalysis.Common": "1.0.1",
     "Microsoft.CodeAnalysis.EditorFeatures.Text": "1.0.1",
     "Microsoft.CodeAnalysis.Workspaces.Common": "1.0.1",
-    "Microsoft.VisualStudio.CoreUtility": "14.3.25407",
-    "Microsoft.VisualStudio.Editor": "14.3.25407",
-    "Microsoft.VisualStudio.Text.Data": "14.3.25407",
-    "Microsoft.VisualStudio.Text.Logic": "14.3.25407",
-    "Microsoft.VisualStudio.Text.UI": "14.3.25407",
-    "Microsoft.VisualStudio.Text.UI.Wpf": "14.3.25407",
+    "Microsoft.VisualStudio.CoreUtility": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Editor": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Text.Data": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Text.Logic": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Text.UI": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Text.UI.Wpf": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Utilities": "15.0.26014-alpha",
     "Microsoft.VisualStudio.OLE.Interop": "7.10.6070",
-    "Microsoft.VisualStudio.Shell.14.0": "14.3.25407",
-    "Microsoft.VisualStudio.Shell.Immutable.10.0": "10.0.30319",
+    "Microsoft.VisualStudio.Shell.Framework": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Shell.15.0": "15.0.26014-alpha",
+    "Microsoft.VisualStudio.Shell.Immutable.10.0": "15.0.26014-alpha",
     "Microsoft.VisualStudio.Shell.Interop": "7.10.6071",
     "Microsoft.VisualStudio.Shell.Interop.8.0": "8.0.50727",
     "Microsoft.VisualStudio.Shell.Interop.9.0": "9.0.30729",
     "Microsoft.VisualStudio.Shell.Interop.10.0": "10.0.30319",
     "Microsoft.VisualStudio.TextManager.Interop": "7.10.6070",
-    "Microsoft.VisualStudio.ComponentModelHost": "14.3.25407-alpha",
+    "Microsoft.VisualStudio.ComponentModelHost": "15.0.26014-alpha",
     "System.Collections.Immutable": "1.1.36",
```

Comment:
I think these should be moved forward to the Dev15 versions, if we are moving the VS references forward. @jmarolf may have an opinion here as well.